asdf_astropy/resources/schemas/slicedwcs/slicedwcs-1.0.0.yaml

```
@@ -0,0 +1,26 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://astropy.org/schemas/astropy/slicedwcs/slicedwcs-1.0.0"
+
+title: Represents the SlicedLowLevelWCS object
+
+description: The SlicedLowLevelWCS class is a wrapper class for WCS that applies slices
+  to the WCS, allowing certain pixel and world dimensions to be retained or
+  dropped.
+  It manages the slicing and coordinate transformations while preserving
+  the underlying WCS object.
+
+allOf:
+  - type: object
+    properties:
+      wcs:
+        tag: "tag:astropy.org:astropy/fits/fitswcs-1*"
+      slices_array:
+        type: array
+        items:
+          - oneOf:
+            - type: integer
+            - type: object
```

```suggestion
            - type: object
```
This needs more details. It appears to be an object with `start` `stop` `step` all either int or None.

---

Sure, I will include it in the schema

---

Thanks! The updated schema looks great.